mandatory/2-function-creation.js

```diff
@@ -25,7 +39,13 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  //return arrayOfStrings.map(string => string.trim().replace("/ ", '').toLowerCase());
+  for (let i = 0; i < arrayOfStrings.length; i++) {
+    arrayOfStrings[i] = arrayOfStrings[i].trim().replace("/", "").toLowerCase();
```

Comment:
This was done nicely. I got stuck here and needed help in applying these array modification.